tutorials/xr/setting_up_xr.rst

```diff
@@ -89,6 +90,24 @@ Next we need to add a script to our root node. Add the following code into this
 		else:
 			print("OpenXR not initialised, please check if your headset is connected")
 
+  .. code-tab:: csharp
+
+    public partial class XRInitializer : Node3D
+    {
+        public override void _Ready()
+        {
+            XRInterface xrInterface = XRServer.FindInterface("OpenXR");
+            if (xrInterface != null && xrInterface.IsInitialized())
+            {
+                GetViewport().UseXR = true;
+
+                GD.Print("OpenXR Initialized Successfully!");
+            }
+            else
+                GD.Print("OpenXR failed init... check headset connection!");
```

Comment:
Use the same message as the GDScript example. Also, our preferred style is to use curly brackets even when not necessary.

```suggestion
            else
            {
                GD.Print("OpenXR not initialized, please check if your headset is connected");
            }
```